### PR TITLE
About Page Layout

### PR DESCRIPTION
**Pull request by chasulin**
- Adds new About page layout with 'alumni' vs 'active' team lists and 'additional advisors' list

### CODE REVIEW

File: src/common/team.js [hunk 1]
```diff
@@ -62,19 +62,54 @@ export default {
...
+  teamActive: [
     {
       name: 'Adnan Pirzada',
       title: 'Software Engineer at Lattice',
       link: 'https://www.linkedin.com/in/adpirz/',
     },
     {
-      name: 'Ali Berlin Johnson',
+      name: 'Alan Fritzler',
+      title: 'Former Lead Designer of Firebase',
```
Comment (goldblatt) [inline src/common/team.js:108]: I don' think this is right

File: src/common/team.js [hunk 2]
```diff
@@ -62,19 +62,54 @@ export default {
...
+      link: 'https://www.linkedin.com/in/alanfritzler/',
+    },
+    {
+      name: 'Alex MacAulay',
       title: 'Former Lead Designer of Firebase',
       link: 'https://www.linkedin.com/in/berlinjohnson/',
```
Comment (goldblatt) [inline src/common/team.js:114]: or this one either

File: src/common/team.js [hunk 3]
```diff
@@ -108,6 +133,11 @@ export default {
       title: 'Executive Business Partner at Pantheon',
       link: 'https://www.linkedin.com/in/brittany-braxton-39187549/',
     },
+    {
+      name: 'Charles Pieper',
+      title: 'Software Engineer',
+      link: 'https://www.linkedin.com/in/chelsi-asulin/',
+    },
```
Comment (goldblatt) [inline src/common/team.js:140]: or this one

File: src/common/team.js [hunk 4]
```diff
@@ -119,20 +149,15 @@ export default {
       link: 'https://www.linkedin.com/in/ckelly90/',
     },
     {
-      name: 'Chris Wilcox',
+      name: 'Claudia Meng',
```
Comment (goldblatt) [inline src/common/team.js:152]: I think this title is wrong

File: src/common/team.js [hunk 5]
```diff
@@ -159,17 +179,17 @@ export default {
       link: 'https://www.linkedin.com/in/dougspan/',
     },
     {
-      name: 'Elwyn Moir',
+      name: 'Erin Kelly',
       title: 'COMET Fellow at Stanford Medicine',
```
Comment (goldblatt) [inline src/common/team.js:183]: I think this title is wrong too

File: src/common/team.js [hunk 6]
```diff
@@ -229,35 +259,50 @@ export default {
       link: 'https://www.linkedin.com/in/asciimike/',
```
Comment (goldblatt) [inline src/common/team.js:259]: I think that mike should probably be in the alumni

File: src/screens/About/About.tsx [hunk 7]
```diff
@@ -11,19 +11,30 @@ import TeamTable from './TeamTable/TeamTable';
...
-    <SidebarLink href="#can">Covid Act Now</SidebarLink>
-    <SidebarLink href="#partners">Our Partners</SidebarLink>
-    <SidebarLink href="#team">The Team</SidebarLink>
-    <SidebarLink href="#model">The Model</SidebarLink>
+    <SidebarLink href="#can">About Covid Act Now</SidebarLink>
+    <SidebarLink href="#partners">Partners</SidebarLink>
+    <SidebarLink href="#founders">Founders</SidebarLink>
+    <SidebarLink href="#advisors">Advisors</SidebarLink>
+    <SidebarLink href="#team">Team</SidebarLink>
+    <SidebarLink href="#model">Model Info</SidebarLink>
```
Comment (mikelehen) [inline src/screens/About/About.tsx:29]: nit: here it says "Model Info" but the actual heading for that section is "Our Model".  Intentional?

File: src/screens/About/About.tsx [hunk 8]
```diff
@@ -11,19 +11,30 @@ import TeamTable from './TeamTable/TeamTable';
...
     <SidebarLink href="#faq">FAQ</SidebarLink>
   </React.Fragment>
 );
 
 const About = ({ children }: { children: React.ReactNode }) => {
+  const [teamList, setTeamList] = useState('active');
+  const teamToShow = teamList === 'alumni' ? TEAM.teamAlumni : TEAM.teamActive;
+  console.log('teamList', teamList);
```
Comment (mikelehen) [inline src/screens/About/About.tsx:37]: intentionally leaving this console.log() in?

File: src/screens/About/TeamTable/TeamTable.style.tsx [hunk 9]
```diff
@@ -1,20 +1,33 @@
 import styled from 'styled-components';
 import Typography from '@material-ui/core/Typography';
 
-export const TeamTableGrid = styled.div``;
-export const TeamMember = styled.div`
+export const TeamTableGrid = styled.div<{
+  isTeam?: Boolean;
```
Comment (mikelehen) [inline src/screens/About/TeamTable/TeamTable.style.tsx:5]: Is `isTeam` used?  If not, please remove from the TeamTable.tsx code that passes them in too?

File: src/screens/About/TeamTable/TeamTable.style.tsx [hunk 10]
```diff
@@ -1,20 +1,33 @@
...
+  display: inline-block;
+
+  @media (min-width: 600px) {
+    column-count: 2;
+  }
+`;
+
+export const TeamMember = styled.div<{
+  isTeam?: Boolean;
+  isAdditionalAdvisors?: Boolean;
```
Comment (mikelehen) [inline src/screens/About/TeamTable/TeamTable.style.tsx:17]: Are `isTeam` and `isAdditionalAdvisors` used?  If not, please remove them from the TeamTable.tsx code that passes them in too?

File: src/screens/About/TeamTable/TeamTable.tsx [hunk 11]
```diff
@@ -1,25 +1,44 @@
...
 import { TeamTableGrid, TeamMember, TeamMemberCell } from './TeamTable.style';
+import {
+  TeamMemberName,
+  TeamMemberTitle,
+} from '../HeadshotGrid/HeadshotGrid.style';
 
 const TeamTable = ({
   people,
+  isTeam,
+  isAdditionalAdvisors,
```
Comment (mikelehen) [inline src/screens/About/TeamTable/TeamTable.tsx:12]: If `<TeamMember>` doesn't actually use `isAdditionalAdvisors` (per my comments above), then I don't think we need it here either?

File: src/screens/About/About.style.tsx [hunk 12]
```diff
@@ -23,3 +23,34 @@ export const Header = styled.div`
...
+  display: flex;
+  justify-content: center;
+  cursor: pointer;
+  font-family: 'Roboto', 'Helvetica', 'Arial', sans-serif;
+  padding: 0.1rem;
+
+  &:first-child {
+    border-right: 1px solid black;
+    background-color: ${({ teamList }) =>
+      teamList === 'active' && 'rgba(59,188,230,.5)'};
```
Comment (mikelehen) [inline src/screens/About/About.style.tsx:48]: nit (feel free to ignore for now): Using magic string values like this is a bit fragile (you could typo the string somewhere by accident) and not self-documenting (there's no easy way to tell what the valid values are).  

It'd be a bit cleaner to use an enum or a string union.

**Enum**
```
enum TeamList {
  Active,
  Alumni,
}
```
Then change `'active'` here to `TeamList.Active`

**String Union**

```
type TeamList = 'active' | 'alumni';
```

and then change the property type above from `teamList?: string` to `teamList?: TeamList`.  And now TypeScript will yell at you if you pass any string other than 'active' or 'alumni'.

File: src/screens/About/About.tsx (hunk 7 above)
Comment (chasulin) [reply src/screens/About/About.tsx:29]: hmm good catch. they were that way in the wireframe but ill just sync them

File: src/screens/About/About.style.tsx (hunk 12 above)
Comment (chasulin) [reply src/screens/About/About.style.tsx:48]: ooo yes. enum it is. thank you!

**Pull request merged**